### PR TITLE
Fix publishing of RID-agnostic packages

### PR DESCRIPTION
**Pull request by jkotas**

### CODE REVIEW

File: eng/pipelines/official/stages/publish.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 parameters:
-  PublishRidAgnosticPackagesFromPlatform: windows_x64
+  PublishRidAgnosticPackagesFromPlatform: browser_wasm
```
Comment (jkotas) [inline eng/pipelines/official/stages/publish.yml]: `windows_x64` leg is not building the packages after #2133. We need to tell the publish process to pick-up the RID-neutral packages from `browser_wasm` leg.

File: eng/pipelines/runtimelab.yml [hunk 2]
```diff
@@ -127,7 +127,7 @@ stages:
         buildArgs: -s clr.alljits+clr.tools+clr.nativeaotlibs+libs+nativeaot.packages -c Release
         extraStepsTemplate: /eng/pipelines/runtimelab/runtimelab-post-build-steps.yml
         extraStepsParameters:
-          uploadIntermediateArtifacts: ${{ variables.isOfficialBuild }}
+          isOfficialBuild: ${{ variables.isOfficialBuild }}
```
Comment (jkotas) [inline eng/pipelines/runtimelab.yml:130]: `isOfficialBuild` flag was not propagated correctly.

**Comment by jkotas** [comment]
cc @yowl

**Comment by yowl** [comment]
Thanks, guess this is another consequence of NativeAOT-LLVM being so far behind.

**Comment by jkotas** [comment]
I think this one is caused more by NativeAOT-LLVM official build flow diverging from dotnet/runtime. I suspect that we will end up redoing #2133 at some point to make the official build flow more similar to dotnet/runtime again.

**Comment by SingleAccretion** [comment]
Does this fix the `win-x64` package publish as well? If it does not and there is more friction to overcome to fix that, then we should revert #2133 and go in the other direction (i. e. build LLVM on the the host platforms, at least for official builds). The reason I thought the current solution better is that it is consistent with the WASM build itself hence easier to understand (and doesn't need to build LLVM twice), but consistency with the runtime in official builds trumps that, even at some complexity overhead, because of the high debugging cost.

**Comment by jkotas** [comment]
Let's give this change a try. If it still does not work, we can re-evaluate the approach.

**Pull request merged**